modules/appeals_api/app/controllers/appeals_api/notice_of_disagreements/v0/notice_of_disagreements/evidence_submissions_controller.rb

```diff
@@ -53,10 +50,9 @@ def create
       )
 
       render status: :created,
-             json: submission,
-             serializer: AppealsApi::EvidenceSubmissionSerializer,
-             key_transform: :camel_lower,
-             render_location: true
```

Comment:
The `AppealsApi::EvidenceSubmissionSerializer` used in decision reviews is different from the others - the HLR, SC, and NOD serializers for decision reviews all use `JSONAPI::Serializer`, but the `AppealsApi::EvidenceSubmissionSerializer` is a plain `ActiveRecord::Serializer` - to keep things consistent, I wrote a new `BaseEvidenceSubmissionSerializer` that uses `JSONAPI::Serializer` like the others. That's where this difference in syntax comes from